airbyte-integrations/connectors/source-google-ads/setup.py

```diff
@@ -7,7 +7,7 @@
 
 # pin protobuf==3.20.0 as other versions may cause problems on different architectures
 # (see https://github.com/airbytehq/airbyte/issues/13580)
-MAIN_REQUIREMENTS = ["airbyte-cdk>=0.51.3", "google-ads==22.1.0", "protobuf", "pendulum"]
+MAIN_REQUIREMENTS = ["airbyte-cdk>=0.51.3", "google-ads==22.1.0", "protobuf", "pendulum>=3.0.0"]
```

Comment:
This needs to be fixed as in my PR if you will merge your first.

Reply:
Done